client/src/components/files/FileCard.vue

```diff
@@ -77,6 +85,13 @@ defineExpose({
   props,
 });
 
+function fileIcon(): string {
```

Comment:
`fileIcon` is a variable name, `getFileIcon()` is a function name.

Reply:
We already have `getFileIcon()` so maybe `showFileIcon` will be better. Technically, the function just show the icon got by the `getFileIcon()` function.